### ISSUE TITLE
the acc constantly decrease

### ISSUE THREAD

**Issue by lichunxiang93** [qjadud1994/YOLOv2-keras#4]
hi,thank you for your share ,while i get problm when i train my data.the loss and the acc are all constantly decrease,do you know why?

**Comment by jubick1337** [comment]
same issue here
